completions/ccache

```diff
@@ -26,8 +29,11 @@ _ccache()
 
     $split && return
 
-    COMPREPLY=( $( compgen -W '$( _parse_help "$1" )' -- "$cur" ) )
-    [[ $COMPREPLY == *= ]] && compopt -o nospace
+    if [[ "$cur" == -* ]]; then
+        COMPREPLY=( $( compgen -W '$( _parse_help "$1" )' -- "$cur" ) )
+        [[ $COMPREPLY == *= ]] && compopt -o nospace
+        return
+    fi
```

Comment:
I had a play, and I think you're right: it does work without this change. I shall update the PR.

I guess the reason that the `if` is needed in the case of `sudo` is because there is another path that could be followed (there is some following code), but not in this case.